cmd/cmd.go

```diff
@@ -86,16 +86,19 @@ func Init(cmd *cobra.Command) (err error) {
 			err = e
 			return
 		}
+		tidbLogCfg := logutil.LogConfig{}
 		if len(slowLogFilename) != 0 {
-			slowCfg := logutil.LogConfig{SlowQueryFile: slowLogFilename}
-			e = logutil.InitLogger(&slowCfg)
-			if e != nil {
-				err = e
-				return
-			}
+			tidbLogCfg.SlowQueryFile = slowLogFilename
 		} else {
 			// Hack! Discard slow log by setting log level to PanicLevel
 			logutil.SlowQueryLogger.SetLevel(logrus.PanicLevel)
+			// Disable annoying TiDB Log.
+			tidbLogCfg.Level = "fatal"
```

Comment:
Perhaps use `"error"` level here, in case of legit errors?

Reply:
TiDB outputs error log randomly even if BR runs successfully. 

By setting level to fatal, we get rid of some of annoying logs, not all. I will add a TODO here.

Reply:
That sounds like a problem of TiDB, but OK.